app/src/main/AndroidManifest.xml

```diff
@@ -66,6 +66,8 @@
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE_LOCATION" />
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
   <uses-permission android:name="android.permission.FOREGROUND_SERVICE_PHONE_CALL" />
+  <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MICROPHONE" />
+  <uses-permission android:name="android.permission.FOREGROUND_SERVICE_CAMERA" />
```

Comment:
Camera permissions shouldn't be a part of the app until functionality is present, otherwise app is going to be rejected by Google Play.

Reply:
Oh ok, gimme a min

Reply:
But one question, I’m using it in the permissions of the foreground service itself, shouldn’t be added also into manifest?

Reply:
Ahh, camera permission, misread, ok, I’ll remove it

Reply:
Done

Reply:
It's still in foregroundServiceType

Reply:
It's still in foregroundServiceType

Reply:
Fixed now